connectors/hub-concur/src/main/java/com/vmware/connectors/concur/controller/HubConcurController.java

```diff
@@ -0,0 +1,121 @@
+/*
+ * Copyright © 2018 VMware, Inc. All Rights Reserved.
+ * SPDX-License-Identifier: BSD-2-Clause
+ */
+
+package com.vmware.connectors.concur.controller;
+
+import com.vmware.connectors.common.payloads.response.Cards;
+import com.vmware.connectors.concur.service.HubConcurService;
+import com.vmware.connectors.concur.util.HubConcurUtil;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.security.core.annotation.AuthenticationPrincipal;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestHeader;
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.bind.annotation.RestController;
+import reactor.core.publisher.Mono;
+
+import javax.servlet.http.HttpServletRequest;
+import java.io.IOException;
+import java.util.Locale;
+
+import static org.springframework.http.MediaType.APPLICATION_FORM_URLENCODED_VALUE;
+import static org.springframework.http.MediaType.APPLICATION_JSON_VALUE;
+
+@RestController
+public class HubConcurController {
+
+    private static final Logger logger = LoggerFactory.getLogger(HubConcurController.class);
+
+    private static final String X_AUTH_HEADER = "X-Connector-Authorization";
+    private static final String X_BASE_URL_HEADER = "X-Connector-Base-Url";
+
+    private final HubConcurService service;
+
+    @Autowired
+    public HubConcurController(HubConcurService service) {
+        this.service = service;
+    }
+
+    /**
+     * API to request cards(pending approvals) for the user
+     * @param username userName of the user logged in
+     * @param vidmAuthHeader Bearer token for authorization
+     * @param baseUrl cocnur base url
+     * @param routingPrefix
+     * @param locale
+     * @return
+     */
+    @PostMapping(
+            path = "/cards/requests",
+            produces = APPLICATION_JSON_VALUE,
+            consumes = APPLICATION_JSON_VALUE
+    )
+    public Mono<Cards> getCards(
+            @AuthenticationPrincipal String username,
+            @RequestHeader(X_AUTH_HEADER) String vidmAuthHeader,
```

Comment:
```vidmAuthHeader``` is not being used.

Reply:
Created APF-1715  to remove unused variables.

Reply:
Wait until APF-1544 is done and then clean up (or just clean up in that ticket).